File: beginner/1116.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
 
    public static void main(String[] args) throws IOException {
		int n;
		Scanner scanner = new Scanner(System.in);
		n = scanner.nextInt();
		
		double x, y;
		
		for(int i = 0 ; i < n ; i ++){
			x = scanner.nextDouble();
			y = scanner.nextDouble();
		
			if( y == 0){
				System.out.println("divisao impossivel");
				continue;
			}
			
			System.out.println(x/y);
		}
 
    }
 
}

File: beginner/1116.c
# include <stdio.h>

int main(){
	
	int n = 0;
	
	scanf("%i", &n);
	int i = 0;
	
	float x,y;
	for(i =0 ; i < n ; ++i ){
		scanf("%f", &x);
		scanf("%f", &y);
		
		if(y == 0)
			printf("divisao impossivel\n");
		else
			printf("%.1f\n", (x/y) );
	}
	
	return 0;
}
